scripts/package

```diff
@@ -23,6 +23,5 @@ DOCKERFILE=package/Dockerfile
 if [ -e ${DOCKERFILE}.${ARCH} ]; then
     DOCKERFILE=${DOCKERFILE}.${ARCH}
 fi
-
-docker build -f ${DOCKERFILE} -t ${IMAGE} .
+DOCKER_BUILDKIT=0 docker build -f ${DOCKERFILE} -t ${IMAGE} .
```

Comment:
https://stackoverflow.com/questions/64221861/failed-to-resolve-with-frontend-dockerfile-v0